catroid/src/main/java/org/catrobat/catroid/cloudmessaging/CloudMessaging.java

```diff
@@ -54,6 +61,30 @@ public void showNotification(CloudMessage cloudMessage) {
 		}
 	}
```

Comment:
Please don't inline the `showNotification` method. The `showNotification` method should be above the `buildNotification`


```java
…some unrelated methods

…more unrelated methods

showNotification()

buildNotification()
```